src/main/java/com/bento909/model/DateMaker.java

```diff
@@ -0,0 +1,49 @@
+package com.bento909.model;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Random;
+import java.text.ParseException;
+
+public class DateMaker {
+    // Generates a date at random
+    public Date dateMaker() {
+        //ArrayList<> month31Days = new List<1,3,5,7,8,10,12>;
+        Date date;
+
+        try {
+            DateFormat df = new SimpleDateFormat("dd-MM-yyyy");
```

Comment:
OK so I have pulled this out of the method, so that it is part of the class declarations.